nncf/tensorflow/quantization/default_quantization.py

```diff
@@ -12,7 +12,11 @@
 """
 from nncf.common.quantization.quantizer_propagation.structs import QuantizationTrait
 from nncf.tensorflow.graph.metatypes import keras_layers as layer_metatypes
+from nncf.tensorflow.graph.metatypes import tf_ops as op_metatypes
 from nncf.tensorflow.graph.metatypes import common
+from nncf.common.graph.operator_metatypes import UnknownMetatype
+
+# If there are no some metatypes it means that they are considered as QuantizationTrait.QuantizationAgnostic
```

Comment:
@kshpv We had the offline discussion that it should be changed in this PR, right?